politeiawww/userwww.go

```diff
@@ -326,6 +353,19 @@ func (p *politeiawww) handleVerifyResetPassword(w http.ResponseWriter, r *http.R
 		return
 	}
 
+	// the trade off here is that we don't want to fail the password change
+	// because we cannot delete all the user's sessions.
+	user, err := p.db.UserGetByUsername(vrp.Username)
+	if err == nil {
+		// log off user everywhere by deleting all sessions
+		err = p.db.SessionsDeleteByUserId(user.ID, "")
+		if err != nil {
+			log.Errorf("SessionsDeleteByUserId() error: %v", err)
+		}
+	} else {
+		log.Errorf("UserGetByUsername() error: %v", err)
```

Comment:
Shouldn't we be calling `RespondWithError()` here instead of logging the error?

Reply:
Good question .. that error means we failed to look up the user which means something is fishy and we should fail the password reset.

Reply:
Adopted your suggestion.